hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -815,30 +816,20 @@ public final Stream<HoodieBaseFile> getLatestBaseFilesInRange(List<String> commi
 
   @Override
   public void loadAllPartitions() {
-    try {
-      readLock.lock();
```

Comment:
This is writing, not reading. Write lock is taken when writing to the map.

Reply:
I only see one usage in `HoodieBackedTableMetadataWriter#getPartitionFileSlicePairs`, we still need to ensure the integrity in case there is a write(sync or reset) or read(with partial update).

Reply:
I don't understand how that is relevant here, the method is just writing to the storage guarded by the lock in this method. The retrieval will happen in a different method.